src/patternfly/patternfly-theme-dark.scss

```diff
@@ -7,3 +7,9 @@ $pf-theme-dark-class: ".pf-theme-dark" !default;
 :where(#{$pf-theme-dark-class}) {
   @include pf-theme-dark($pf-theme-dark-class);
 }
+
+/* stylelint-disable */
+.ws-preview,
+.ws-example {
```

Comment:
Maybe also add `.ws-code` here? So the doc will look nicer.

Reply:
Oh I'm just adding these temporarily and was planning on taking them out. Ideally I think we would put dark theme workspace changes in the patternfly-org repo where the workspace/website theme styles are hosted.